docker/luna_tutorial/dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:18.04
+
+## supress interactive dialogs
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG USER_ID
+ARG GROUP_ID
+ARG USER_NAME
+ARG GROUP_NAME
+RUN echo $USER_ID:$GROUP_ID $USER_NAME:$GROUP_NAME
+
+## install OS dependencies
+RUN apt-get update
+RUN apt-get install software-properties-common openslide-tools python-openslide default-jre git vim curl tree -y
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt-get install python3.9 python3-pip -y
+
+
+## create user so python packages are not being installed with root and jupyter is not executed with rootr
+## map host user to container user so files can be easily accessed from host and container
+RUN groupadd $GROUP_NAME
+RUN useradd -l -rm -d /home/$USER_NAME -s /bin/bash -g $GROUP_NAME -u $USER_ID $USER_NAME
+USER $USER_NAME
+WORKDIR /home/$USER_NAME
+ENV PATH=$PATH:/home/$USER_NAME/.local/bin
+
+## install luna
+# applying temp hack for getting pyspark v3.1.2 to play well with deltalake
+# tensorboard and sklearn are missing pyluna dependencies
+RUN python3 -m pip install --no-cache-dir --user --upgrade pip \
+	&& pip3 install pillow>=8.1.1 pyluna pyluna_pathology \
+	&& pip3 uninstall -y pyspark \
+	&& pip3 install pyspark==3.1.2 \
+	&& pip3 install tensorboard sklearn
+
+## install jupyter
+RUN python3 -m pip install --no-cache-dir jupyter
+
+## set env vars
+ENV LUNA_HOME="/home/$USER_NAME/luna"
+# pyspark adds /bin/java
+ENV JAVA_HOME="/usr"
+ENV LANG="C.UTF-8"
+ENV PYSPARK_PYTHON="/usr/bin/python3"
+ENV PYSPARK_DRIVER_PYTHON="/usr/bin/python3"
+
+
+## verify installation
+RUN java --version
+RUN echo $JAVA_HOME
+RUN echo $LUNA_HOME
+RUN echo PYSPARK_PYTHON
+RUN echo PYSPARK_DRIVER_PYTHON
```

Comment:
These will also add layers, may want to trim this down, or leave them as comments? if they are only here for debugging purposes. Missing $ in $PYSPARK_PYTHON $PYSPARK_DRIVER_PYTHON